code/modules/jobs/job_types/chief_engineer.dm

```diff
@@ -48,8 +48,9 @@
 	ears = /obj/item/radio/headset/heads/ce
 	uniform = /obj/item/clothing/under/rank/engineering/chief_engineer
 	shoes = /obj/item/clothing/shoes/sneakers/brown
+	glasses = /obj/item/clothing/glasses/meson/engine
 	head = /obj/item/clothing/head/hardhat/white
-	gloves = /obj/item/clothing/gloves/color/black
+	gloves = /obj/item/clothing/gloves/color/yellow
```

Comment:
```suggestion
	gloves = /obj/item/clothing/gloves/color/black
```

Don't mix unrelated changes.

Reply:
It's related to door hacking being easier for engineers, but alright I'll make an entirely separate PR for the CE gloves.